tools/site_and_regional/run_neon.py

```diff
@@ -150,99 +138,52 @@ def get_parser(args, description, valid_neon_sites):
+                default = '0Y')
 
-    tr_parser.add_argument('--start-year',
+    parser.add_argument('--start-date',
                 help='''           
-                Start year for running CTSM simulation.
+                Start date for running CTSM simulation in ISO format.
                 [default: %(default)s]
                 ''',               
                 action="store", 
-                dest="start_year",                                                                                                                                                 
+                dest="start_date",                                                                                     
                 required = False,
-                type = int,        
-                default = 2018)
+                type = datetime.date.fromisoformat,
+                default = datetime.datetime.strptime("2018-01-01",'%Y-%m-%d'))
 
-    tr_parser.add_argument('--end-year',
+    parser.add_argument('--end-date',
                 help='''
-                End year for running CTSM simulation.
+                End date for running CTSM simulation in ISO format.
                 [default: %(default)s]
                 ''', 
                 action="store",
-                dest="end_year",
+                dest="end_date",
                 required = False,
-                type = int,
-                default = 2020)
-
-    #parser.add_argument('--spinup',
-    #            help='''
-    #            AD spin-up
-    #            [default: %(default)s]
-    #            ''', 
-    #            action="store_true",
-    #            dest="ad_flag",
-    #            required = False,
-    #            default = True)
-    #parser.add_argument('--postad',
-    #            help='''
-    #            Post-AD spin-up
-    #            [default: %(default)s]
-    #            ''', 
-    #            action="store_true",
-    #            dest="postad_flag",
-    #            required = False,
-    #            default = True)
-    #parser.add_argument('--transient',
-    #            help='''
-    #            Transient
-    #            [default: %(default)s]
-    #            ''', 
-    #            action="store_true",
-    #            dest="transient_flag",
-    #            required = False,
-    #            default = True)
-
-    #parser.add_argument('--sasu','--matrix',
-    #            help='''
-    #            Matrix (SASU) spin-up
-    #            [default: %(default)s]
-    #            ''', 
-    #            action="store_true",
-    #            dest="sasu_flag",
-    #            required = False,
-    #            default = False)
+                type = datetime.date.fromisoformat,
+                default = datetime.datetime.strptime("2020-01-01",'%Y-%m-%d'))
+
+    parser.add_argument('--run-from-postad',
+                        help='''
+                        For transient runs only - should we start from the postad spinup or finidat?
```

Comment:
I'm not sure what this note is asking, but a `run-from-postad` the initial conditions should be from the postad simulation, as opposed to `run-from-latest` which would run from the last restart file of a transient run that's available.

Reply:
By default the transient run uses the finidat file found on the s3 server.  But if you want to redo the spin-up and rerun from the postad you would use this flag to trigger that.

Reply:
maybe some a comment on the code to this effect would help clarify